### ISSUE TITLE
Usage with vanilla puppeteer?

### ISSUE THREAD

**Issue by mon-jai** [ulixee/unblocked#75]
Can the this repository be used as Puppeteer plugins, similar to how we use `puppeteer-extra-plugin-stealth`?

```js
// puppeteer-extra is a drop-in replacement for puppeteer,
// it augments the installed puppeteer with plugin functionality
const puppeteer = require('puppeteer-extra')

// add stealth plugin and use defaults (all evasion techniques)
const StealthPlugin = require('puppeteer-extra-plugin-stealth')
puppeteer.use(StealthPlugin())
```

---

It would be good if we could use unblocked with puppeteer's API

**Comment by blakebyrnes** [comment]
I never responded to this. Unfortunately, it would be possible to do, but would require a lot of work. This entire project works around puppeteer's limitations in scraping, which includes going to the protocol level of the platform. To really get there, we'd have to re-implement all of puppeteers apis on top of the current stack. That's not a goal of the project at the moment, so I'm going to mark this as won't fix. If you are wanting to take this on, I think the approach would be to create it with plugins and identify areas that need to be added to plugins to make it possible.
